package-builder/build-manifest.nix

```diff
@@ -64,6 +65,9 @@ pkgs.runCommandNoCC name
             ${pkgs.lib.optionalString pkgs.stdenv.isDarwin ''
               signDarwinBinariesInAllOutputs
             ''}
+            ${pkgs.lib.optionalString pkgs.stdenv.isLinux ''
+              autoPatchelfPostFixup
```

Comment:
I'm still catching up on what this fixes...I see in the commit message that the interpreter is patched, but since it still resolves the same way in all the situations you showed, why do we care?

Reply:
It provides the slim guarantee that the interpreter/linker (ld-linux-aarch64) comes from a consistent place but I'm not sure that it makes any practical difference. We can't do what the ticket originally asked for, which is to find store path equivalents of any system libraries that an impure build might have linked against.